sprint7/encyklopedia1.js

```diff
@@ -0,0 +1,43 @@
+/*Constructor function
+Słowo kluczowe “this” w kontekście funkcji*/
+
+/*
+Funkcję Constructor function tworzymy w celu nowe obiekty dla uzytkowników aplikacji,
+kont bankowych, samochodów lub cokolwiek innego, które mają te same właściwości i metody.
```

Comment:
To zdanie nie jest po polsku. `Funkcję Constructor function tworzymy w celu ...` - tutaj można napsiać np. tworzenia, przypisywania czegoś do czegoś.